scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#270.sql

```diff
@@ -0,0 +1,16 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- Fix #270: Invalid surgeries in database
+DELETE FROM surgery
+WHERE surgery.id IN (2235874,7656953,3084781,56773449,2854022,43669707,4479617,4462790,4462414,128161513,43669899,64323554,117548993,56773602,63356896,100621556,100621549,92990592,100618725,111194593,111194624);
+
+---- Test results
+SELECT s.transmitter_id
```

Comment:
it won't actually affect the script operation, but we may need to comment out this select as part of the test results